blogs/contractionorder.md

```diff
@@ -63,7 +98,15 @@ In practice, there are many tools for tensor network contraction order optimizat
 * [OMEinsumContractionOrder.jl](https://github.com/TensorBFS/OMEinsumContractionOrders.jl): an open-source Julia package for finding the optimal contraction order of tensor networks, is used as backend of [OMEinsum.jl](https://github.com/under-Peter/OMEinsum.jl) and [ITensorNetworks.jl](https://github.com/mtfishman/ITensorNetworks.jl).
 * [Cotengra](https://cotengra.readthedocs.io/en/latest/)[^Gray] : a python library for contracting tensor networks or einsum expressions involving large numbers of tensors.
 
-Various methods have been proposed for optimizing the contraction order, here we introduce some of them:
+Various methods have been proposed for optimizing the contraction order, as shown in the table below, here we introduce some of them.
+
+| Optimizer      | Description | Available in |
+| ----------- | ----------- | ----------- |
+| Exhaustive Search | Slow, exact | TensorOperations.jl |
+| Greedy Algorithm | Fast, heuristic | OMEinsumContractionOrders.jl, Cotengra |
+| Binary Partition | Fast, heuristic | OMEinsumContractionOrders.jl, Cotengra |
+| Local Search | Fast, heuristic | OMEinsumContractionOrders.jl |
+| Exact Treewidth | Slow, exact | OMEinsumContractionOrders.jl |
 
 #### Exhaustive Search
 
```

Comment:
In the "breadth-first" explanation below, I think two instances of "constructing" are supposed to be "contracting", i.e.

the breadth-first method construct the set of intermediate tensors by 

constructing->contracting? 

$c$ tensors ($c \in [1, n - 1]$, where $n$ is the number of tensors) in each step, and record the optimal cost for constructing each intermediate tensor. Then in the last step, the optimal cost for
 
constructing->contracting?

all $n$ tensors is obtained.